tests/core/data_layer/test_data_rpc.py

```diff
@@ -22,16 +22,22 @@
 from chia.wallet.wallet_node import WalletNode
 from tests.setup_nodes import setup_simulators_and_wallets, bt
 from tests.time_out_assert import time_out_assert
+from tests.util.socket import find_available_listen_port
 from tests.wallet.rl_wallet.test_rl_rpc import is_transaction_confirmed
 
 pytestmark = pytest.mark.data_layer
 nodes = Tuple[WalletNode, FullNodeSimulator]
 
 
-async def init_data_layer(root_path: Path) -> AsyncIterator[DataLayer]:
-    test_rpc_port = uint16(21529)
+async def init_data_layer(root_path: Path, wallet_rpc_port: int) -> AsyncIterator[DataLayer]:
     config = bt.config
```

Comment:
I guess I'm still confused about this. what's `bt` here? it's not the fixture, right?

I would have expected it to be more reasonable to configure the service here, by updating `config`. If `bt` is the fixture, it will have a root path configured already too.